frontend/src/components/OperationsTableCard/OperationsTableCard.tsx

```diff
@@ -143,6 +143,24 @@ const OperationsTableCard: FunctionComponent<OperationsTableCardProps> = (props)
     props.history.push(`/queries/build/${operation.get('id') as number}/`);
   };
 
+  const onUpdateOperation = (operation: OperationMap) => {
```

Comment:
I see this only being used once ... inline it. i.e. instead of creating 3 lines of code just to call one function, call the function where it needs to be used.